pkgs/applications/misc/gnuradio/default.nix

```diff
@@ -28,9 +28,11 @@ stdenv.mkDerivation rec {
   name = "gnuradio-${version}";
   version = "3.7.11";
```

Comment:
Please update version attribute.

Reply:
👍 
I also changed it to use fetchFromGithub